src/utilities/utilities.ts

```diff
@@ -79,8 +79,15 @@ export async function execFile(
         `Exec: ${executable} ${args.join(" ")}`,
         folderContext.name
     );
-    if (configuration.runtimePath.length > 0 && customSwiftRuntime) {
-        options.env = { ...options.env, ...swiftRuntimeEnv(options.env) };
+    if (customSwiftRuntime) {
+        const runtimeEnv = swiftRuntimeEnv(options.env);
+        if (runtimeEnv && Object.keys(runtimeEnv).length > 0) {
+            options.env = { ...options.env, ...runtimeEnv };
+        }
+    }
+    // it options contain environment keys add them to process environment keys
```

Comment:
Not spelling
```suggestion
    // if options contain environment keys add them to process environment keys
```

Reply:
This code got deleted so no need to fix comment. It is now setup the same as in `execFileStreamOutput`